docs/source/resources.rst

```diff
@@ -6,7 +6,9 @@ Resources
 Videos
 ------
 
-- J. Fumero, `TornadoVM: Transparent Hardware Acceleration for Java…and Beyond! <https://www.youtube.com/watch?v=HxhO4hMKi5M>`__, Devoxx Ukraine 2021. 
+- A. Stratikopoulos, `Write Once, Run Anywhere... Well, What About Heterogeneous Hardware? <https://fosdem.org/2023/schedule/event/hardware/>`__, FOSDEM 2023 
+
+- J. Fumero, `TornadoVM: Transparent Hardware Acceleration for Java and Beyond! <https://www.youtube.com/watch?v=HxhO4hMKi5M>`__, Devoxx Ukraine 2021. 
```

Comment:
the names in GitHub seems to be breaking. The first capital of the names are replaced by `i` and `x`.

Reply:
okay, this was already like this. It is because the MD is inconsistent with the format for the documentation [site](https://tornadovm.readthedocs.io/en/latest/index.html).